.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
As a heads up we only included a settings.json in preview here

Reply:
I'll double check if we need this one as well. There is a launch.json in Elements